.coveragerc

```diff
@@ -7,3 +7,6 @@ omit =
     *_test.py
     *core/tests/*
     *__init__.py
+    # TODO(lilithxxx): Remove python_utils from the list once the codebase is
+    # run under Python 3.
+    *python_utils.py
```

Comment:
Do you actually need the * at the beginning? (If so please still add a / after it so that we don't overly-match filenames like xxx_python_utils.py.)

Reply:
Removed the * at beginning